ui/src/shared/components/cells/Cells.tsx

```diff
@@ -65,16 +80,24 @@ class Cells extends Component<Props & WithRouterProps> {
   }
 
   private get cells(): Layout[] {
+    const {state} = this.props
     return this.props.cells
       .filter(c => c.status === RemoteDataState.Done)
-      .map(c => ({
-        ...c,
-        x: c.x,
-        y: c.y,
-        h: c.h,
-        w: c.w,
-        i: c.id,
-      }))
+      .map(c => {
+        const view = getByID<View>(state, ResourceType.Views, c.id)
+        let cell = {
```

Comment:
this should be a `const`

Reply:
It cants be, because we edit it below

Reply:
an obj (and array) can be explicitly reassigned, even as a const.
```
const obj = {a: 1}
console.log(obj.a) // 1
obj.a = 2
console.log(obj.a) // 2
obj.b = 3
console.log(obj) // {a: 2, b: 3}
```